package.json

```diff
@@ -15,6 +15,11 @@
     "example_4:watch": "npm run build && cd examples && npm run example_4:watch",
     "test": "npm run build && npm run mocha && npm run examples"
   },
+  "importSort": {
+    ".js": {
+      "style": "module"
```

Comment:
I'd never seen the import sort package before, or this module style, but I quite like its logic. May have to add this in other projects haha. Thanks!

Reply:
Was on a project where different IDEs had different ideas on import sort order and formatting. This did away with those issues :-) It should match up with IntelliJ/Webstorm's sort order.